pkg/virt-launcher/virtwrap/api/converter.go

```diff
@@ -810,6 +818,13 @@ func Convert_v1_VirtualMachine_To_api_Domain(vmi *v1.VirtualMachineInstance, dom
 					Template: filepath.Join(c.OVMFPath, EFIVarsSecureBoot),
 				}
 			} else {
+                                _, err1 := os.Stat(filepath.Join(c.OVMFPath, EFICode))
```

Comment:
Why not do the same thing for the secureboot case?

Reply:
Do you see the checking the EFI-secboot branch is of any practical use? 
I had the code but hesitated to put into the PR.

Reply:
I do not see that branch in your fork.

Reply:
In my local repo, running a test on it, will check in a moment.